flash.sh

```diff
@@ -318,6 +318,113 @@ flash_gaia()
 	return $?
 }
 
+flash_rpi2()
+{
+	echo Notice: This script will create partitions and
+	echo flash images on the SD if the host machine is Linux.
+	echo On Mac will flash images only. You have to create
+	echo the partitions on the SD manually if the host
+	echo machine is Mac OS X.
+
+	DRIVE=$1
+
+	if [ -z "${PARTITION_NAME}" ]; then
+		echo You must specify a SD card drive
+	fi
+
+	sudo dd if=/dev/zero of=$DRIVE bs=1024
+
+	sed -e 's/\t\([\+0-9a-zA-Z]*\)[ \t].*/\1/' << EOF | fdisk ${DRIVE}
+	  o
+	  n
+	  p
+	  1
+
+	  +48M
+	  n
+	  p
+	  2
+
+	  +537M
+	  n
+	  p
+	  3
+
+	  +128M
+	  n
+	  p
+	  4
+
+
+	  a
+	  1
+	  p
+	  w
+	  q
+	EOF
+
+	mkfs.vfat -F 32 -n "boot" ${DRIVE}p1
+	mkfs.ext4 -L "system" ${DRIVE}p2
+	mkfs.ext4 -L "cache" ${DRIVE}p3
+	mkfs.ext4 -L "userdata" ${DRIVE}p4
+
+	if [ -e "out/target/product/$DEVICE/boot.img" ]; then
```

Comment:
We should support the way ./flash.sh can be called and not always flash everything